CMakeLists.txt

```diff
@@ -177,11 +176,11 @@ if(("${BUILD_TYPE_UPPER}" STREQUAL "DEBUG") OR ("${BUILD_TYPE_UPPER}" STREQUAL "
     option(ENABLE_BUILD_TESTS "Build tests" ON)
     option(ENABLE_VALGRIND_TESTS "Build tests with valgrind" ON)
     # TODO enable when the internal docs ready
-    set(INTERNAL_DOCS "Include developers notes and internal API description into generated Documentation" OFF) 
+    set(INTERNAL_DOCS NO)
```

Comment:
It was bugfix, because the `set()` expects the variable's value as the second parameter. It is not an option because it is (or it was, so far) strictly connected with the build type - enabled in Debug, disabled otherwise. I'm not strictly against making it a cmake option. Recently, I've set the variable always to OFF (should be NO) because the internal documentation wasn't built even if the variable was ON - it is not ready in the build system yet (internal headers are not parsed), so I wanted to enable it when ready.